src/main/java/nl/tudelft/pixelperfect/gamestates/PlayState.java

```diff
@@ -76,7 +76,7 @@ public void updateMovement(float tpf) {
       observer.rotate(0, -0.75f * tpf, 0);
     }
     if (game.isDebugTrigger()) {
```

Comment:
I believe this is not the right way to do the event handling. Currently, when I hold down the correspondig button, the debug mode is flipped at a very high frequency. There must be some way to register a callback as listener to a key press or key release event, provided by the jMonkeyEngine.